static/gen/ast_defs.pony

```diff
@@ -157,20 +157,19 @@ primitive ASTDefs
       .> has("block", "Expr")
     
     for name in [
-      "Not", "Neg", "NegUnsafe", "AddressOf", "DigestOf"
+      "Not"; "Neg"; "NegUnsafe"; "AddressOf"; "DigestOf"
     ].values() do
       g.def(name)
         .> in_union("UnaryOp", "Expr")
         .> with_type()
         .> has("expr", "Expr")
     end
     
-    for name in [
-      "Add", "AddUnsafe", "Sub", "SubUnsafe",
-      "Mul", "MulUnsafe", "Div", "DivUnsafe", "Mod", "ModUnsafe",
-      "LShift", "LShiftUnsafe", "RShift", "RShiftUnsafe",
-      "Is", "Isnt", "Eq", "NE", "LT", "LE", "GE", "GT", "And", "Or", "XOr"
-    ].values() do
+    for name in ["Add"; "AddUnsafe"; "Sub"; "SubUnsafe"; "Mul"
+                 "MulUnsafe"; "Div"; "DivUnsafe"; "Mod"
+                 "ModUnsafe"; "LShift"; "LShiftUnsafe"; "RShift"
+                 "RShiftUnsafe";"Is"; "Isnt"; "Eq"; "NE"; "LT"
+                 "LE"; "GE"; "GT"; "And"; "Or"; "XOr"].values() do
```

Comment:
Sorry to be a pest, but this style is still different from the original - the line breaks are in different places, and the indentation is changed.

I'd prefer to just change the commas to semicolons, without also changing other things. If you don't want to invest any more time in this, I'll happily take care of it tonight in my own commit.